dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java

```diff
@@ -116,7 +114,9 @@ public class TeradataLogsConnector extends AbstractTeradataConnector implements
         "L.SessionWDID",
         "L.Statements",
         "L.TotalIOCount",
-        "L.WarningOnly"
+        "L.WarningOnly",
+        "ST.SQLRowNo",
+        "ST.SQLTextInfo"
```

Comment:
Why the order change? It seems like this should have been a 1 line change, not a 7 line change?

Reply:
Done. Reorder is not significant and, apparently, requires enum updates which I definitely didn't intend.